pkg/resource/google/google_storage_bucket_iam_binding.go

```diff
@@ -0,0 +1,19 @@
+package google
+
+import "github.com/cloudskiff/driftctl/pkg/resource"
+
+const GoogleStorageBucketIamBindingResourceType = "google_storage_bucket_iam_binding"
+
+func initGoogleStorageBucketIamBindingMetadata(resourceSchemaRepository resource.SchemaRepositoryInterface) {
+	resourceSchemaRepository.SetNormalizeFunc(GoogleStorageBucketIamBindingResourceType, func(res *resource.Resource) {
+		res.Attributes().SafeDelete([]string{"force_destroy"})
+		res.Attributes().SafeDelete([]string{"etag"})
+	})
+	resourceSchemaRepository.SetResolveReadAttributesFunc(GoogleStorageBucketIamBindingResourceType, func(res *resource.Resource) map[string]string {
+		return map[string]string{
+			"bucket": *res.Attrs.GetString("bucket"),
+		}
+	})
+	resourceSchemaRepository.SetFlags(GoogleStorageBucketIamBindingResourceType, resource.FlagDeepMode)
```

Comment:
either this resource shouldn't have the deep mode, or you forgot to add a details fetcher for it

Reply:
no it doesn't need the details fetcher the details are all already fetched on enumeration. And calling the readresource remove the member list for a reason I couldn't find

Reply:
I can't find any docs about that resource, is that related to a particular version of the provider ?

Reply:
Ok figured out it's named resource_iam_binding in the provider's code https://github.com/hashicorp/terraform-provider-google/blob/d696c5ed2fb052896b29a6502f1f7b3b2d1c650d/google/resource_iam_binding.go#L65

Reply:
yeah there is se same 3 kind of iam resources for plenty of service so they made a generic way to handle them.
But still I made the correct details fetcher and it didn't give the member list. And as we already have everything I removed it...

Reply:
You forgot to add the `role` attribute to read attributes (see https://github.com/hashicorp/terraform-provider-google/blob/master/google/resource_iam_binding.go#L281-L291), I tested on your branch and it no longer drifts with the common details fetcher